docs/faq.md

```diff
@@ -115,7 +115,8 @@ We're looking to support:
 Referenced throughout this repo, the term "UWP" references a
 [Universal Windows Platform App](https://docs.microsoft.com/en-us/windows/uwp/get-started/) -
 an app running inside an app container, at a lower privilege level, with
-package identity, delivered via an MSIX. This app's lifecycle from activation
+package identity, delivered via an MSIX. It is the only app type supported on Windows devices 
+such as Xbox, HoloLens and Surface Hub (Windows 10 Team). This app's lifecycle from activation
```

Comment:
Two remarks:
* I did not list Windows 10 IoT Core here since to my understanding that product is [currently in the process of being superceded by Windows 10 IoT Enterprise](https://blogs.windows.com/windowsdeveloper/2020/05/26/build-your-iot-devices-with-windows-for-iot-a-comprehensive-platform-for-every-device-developer/) - which supports Win32 apps. I am also not yet listing Windows 10X here since I don't yet know about the exact development details Windows 10X will arrive with.
* Windows 10 Team now supports apps like the new Edge browser based on Chromium or Microsoft Teams (see [here](https://docs.microsoft.com/en-us/surface-hub/surface-hub-2020-update-whats-new#additional-features)) which to my understanding aren't UWP apps even on the Surface Hub so that wording might be a bit imprecise ("only app type supported"). That said, at least commonly [UWP apps are the only ones being supported](https://techcommunity.microsoft.com/t5/surface-it-pro-blog/new-surface-hub-os-update-released-for-public-preview/bc-p/1552149/highlight/true#M313) on Windows 10 Team so the wording might be fine after all.

Reply:
Please move the "It is the only type..." to a separate paragraph. This paragraph should describe precisely what a UWP _is_ and put the _where it's useful_ separately.